tgui/packages/tgui/interfaces/TeleConsole/types.ts

```diff
@@ -18,6 +18,13 @@ export interface TeleConsoleData {
   padNum: number;
   maxBookmarks: number;
   bookmarks: BookmarkData[];
+  destinations: LongRangeData[];
+  disk: BooleanLike;
+}
+
+export interface LongRangeData {
+  ref: string;
```

Comment:
Doesn't look like you're using the `ref`, instead passing the `name` back in places. Given that names should be unique here that's fine, but as you're giving it the data you'd might as well use it.

Prefer `fooRef` as a variable name here, as `ref` has meaning within React and can complicate trying to pass it around. (Not a problem with how I think you'd use it, but avoids an issue and makes it easier for the next person to use this as an example)